### DISCUSSION TITLE
在转移文件后，打开alas一直提示检查deploy。yaml设置。重装也是

### DISCUSSION THREAD

**Discussion by jarviexu** [LmeSzinc/AzurLaneAutoScript#2710] [Q&A]
![微信图片_20230603194044](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/135429995/20451812-6e46-4297-912c-be21391826f4)

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/2687#issuecomment-1572526474
